astropy/coordinates/tests/test_funcs.py

```diff
@@ -33,6 +33,38 @@ def test_sun():
     gcrs2 = get_sun(Time([northern_summer_solstice, equinox_2, northern_winter_solstice]))
     assert np.all(np.abs(gcrs2.dec - [23.5, 0, -23.5]*u.deg) < 1*u.deg)
 
+def test_sun_02():
+    """
+    Test `that get_sun` produces similar values to skyfield (de421).
```

Comment:
A minor nitpicky point: Most probably this link wouldn't work. Either you can put it in double backticks \`\`get_sun\`\` or link the whole path. We don't generate docs for the tests, so this is super minor and won't result a travis/build/etc fail.